<launch>

  <arg name="worldfile" default="$(find exploration_ros)/stagefiles/hospital.world"/>

  <master auto="start"/>
  <param name="/use_sim_time" value="true"/>

  <remap from="scan" to="base_scan"/>


<node pkg="exploration_ros" name="slam" type="slam_node" output="screen" args="-timeUpdate 0 -angularUpdate 0.5 -linearUpdate 0.5 -scanTopic scan -type 0" />


<!--
<include file="$(find exploration_ros)/move_base_config/slam_gmapping.xml"/>
-->

<node pkg="exploration_ros" name="planner" type="frontier_planner" output="screen" args="-iter -1"  />


<node pkg="stage_ros" type="stageros" name="stageros" args="$(arg worldfile)" respawn="false" required="true" >
<param name="base_watchdog_timeout" value="0.2"/>
</node>


<node pkg="rviz" type="rviz" name="visualizer" output="screen" args="-d $(find exploration_ros)/config.rviz"/>


<include file="$(find exploration_ros)/move_base_config/move_base.xml"/>



</launch>

